reference/marketing.v2.yml

```diff
@@ -2,61 +2,37 @@ swagger: '2.0'
 info:
```

Comment:
remove the changes to this file from the pr.

Reply:
This change also invalidates the file.